### ISSUE TITLE
How does one "update the SDK to which it belongs"

### ISSUE THREAD

**Issue by mgwalm** [NuGet/docs.microsoft.com-nuget#870]
Ive searched all over the docs and on GitHub and no where does it tell onehow to update the sdk. 
Also the learn more link only takes one to this page which doesn't give any more info than the ide!

Please updater this docs to tell one how to update the sdk.

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 108d587b-429e-3459-5698-3b40e44f4c3e
* Version Independent ID: 76ad6a3f-4dde-0883-1df9-0f173f0ca7e7
* Content: [NuGet Package Manager UI Reference](https://docs.microsoft.com/en-au/nuget/tools/package-manager-ui#implicit_reference)
* Content Source: [docs/tools/Package-Manager-UI.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/tools/Package-Manager-UI.md)
* Service: **nuget**
* GitHub Login: @kraigb
* Microsoft Alias: **kraigb**

**Comment by kraigb** [comment]
Because it's generic message, you have to infer the containing SDK from the context. For example, if you see it for .NET Core or .NET Standard, then update your .NET Core SDK. There isn't much more the docs can really say to this point; even with .NET Core, it's something that you might have installed independently, or perhaps through the VS installer.

The point of the message is simply that the Package Manager UI cannot itself do an update on that package. There may not, in fact, be any update to consider, thus the message to update the containing SDK.

In any case, I've added a little bit more to the article.

**Comment by mgwalm** [comment]
I understand that, however, please provide an answer to what I asked - How does one update the SDK?

**Comment by kraigb** [comment]
That depends on the SDK in question. You need to find that SDK's website and obtain whatever download/installer is available. For .NET/.NET Core, for example, you'd go to https://www.microsoft.com/net/download/.

**Comment by mgwalm** [comment]
Thanks.  Why wasn’t that link on the web page.

 

More to the point it should be in the ide at the appropriate point. Ie, where it says read more. Which doesn’t give one any more information than in  the ide.

**Comment by kraigb** [comment]
A specific link isn't in the product because the message is generic and not specific to .NET. As for noting it in the docs, I'd recommend creating another issue for that. I no longer work on the NuGet docs, so a new issue is the best way to make the request (or submit a PR directly).

**Comment by dotnetchris** [comment]
I don't agree that this issue is actually resolved, i was about to open this exact same issue.

**Comment by cthames** [comment]
I agree with @dotnetchris . It isn't clear. 
.NET Core 2.2.6 is installed from  https://www.microsoft.com/net/download/
However 2.2.6 is still not selectable (see image below)
It does say "blocked by project" and "unavailable due to additional constraints in the project or packages.config". However the only thing mentioning 2.2 in the project file is the following, are we supposed to manually edit the project file and update it to `netcoreapp2.2.6` or is there a more appropriate means as this isn't the only project in the solution.
```
  <PropertyGroup>
    <TargetFramework>netcoreapp2.2</TargetFramework>
  </PropertyGroup>
```
![image](https://user-images.githubusercontent.com/1069926/62226356-b88d5780-b37f-11e9-8210-07bf0e88e15d.png)

**Comment by rrelyea** [comment]
Here is the documentation on how to download and install newer .NET Core SDK versions.
https://docs.microsoft.com/en-us/dotnet/core/sdk

**Comment by cthames** [comment]
@rrelyea the issue isn't with the installation of the SDK, it's with **updating/upgrading** a current solution/project with a newer version of .NET Core SDK. e.g. 2.2.0 --> 2.2.6

In the link provided
There is a link labeled "Select the .NET Core version to use " that goes to https://docs.microsoft.com/en-us/versions/selection.md which is a `404 - Page not found` with unhelpful suggested links.

Like mentioned before everything about updating a SDK (in a project / solution), especially the .NET Core is completely unclear.

**Comment by rrelyea** [comment]
Let's say you installed net core sdk 2.2.
Then you should upgrade the netcoreapp TFM from netcoreapp2.1 to netcoreapp2.2.
As new runtime versions (patches) are release, in 2.2.x era, a new SDK is released.

Understanding exactly how net core project suggest that you get the latest patched version is something the nuget team doesn't own. 
I would imagine that it is, or should be covered in these docs: https://docs.microsoft.com/en-us/dotnet/core/

Please engage via issues there.

**Comment by cthames** [comment]
Will do. Confusing since this (see image) is all within the VS NuGet Package Manager.
![image](https://user-images.githubusercontent.com/1069926/62241506-e59d3280-b39e-11e9-9f5d-4cdbee4827fa.png)

**Comment by rrelyea** [comment]
Understood. In netcore 3, that won't be true anymore.
There will be no packagereferences (even implicitly added ones like that) to get Net Core framework assemblies.

So hopefully, that will improve several aspects of Net Core usage.
